src/arduino-rfm/lorawan-arduino-rfm.cpp

```diff
@@ -242,58 +229,80 @@ void LoRaWANClass::setDevAddr(const char *devAddr_in)
 
     //Reset frame counter
     Frame_Counter_Tx = 0x0000;
-
-    //Reset RFM command status
-    RFM_Command_Status = NO_RFM_COMMAND;
 }
 
 void LoRaWANClass::setDeviceClass(devclass_t dev_class)
 {
     LoRa_Settings.Mote_Class = (dev_class == CLASS_A)? CLASS_A : CLASS_C;
```

Comment:
CLASS_A or CLASS C? or hexadecimal?? 🤔

Reply:
Is the same, but i should be using only the CLASS_A or CLASS_C typos